pkgs/by-name/nr/nrc-wrapper-go/package.nix

```
@@ -0,0 +1,31 @@
+{
+  fetchFromGitHub,
+  lib,
+  buildGoModule,
+}:
+buildGoModule rec {
+  pname = "nrc-wrapper-go";
+  version = "v1.3.8-2";
+
+  src = fetchFromGitHub {
+    owner = "technicfan";
+    repo = "nrc-wrapper-go";
+    rev = version;
+    hash = "sha256-LNLnh3Vnr4OKIXbTg5HgOKopkchKP8XS/eTw2tBe20I=";
+  };
+
+  vendorHash = "sha256-PvwvVavGTV5QGprMn1M2SOq6upaYCzyzM6dNvZzlXo4=";
+
+  postInstall = ''
+    mv $out/bin/main $out/bin/nrc-wrapper
+  '';
+
+  meta = with lib; {
+    description = "NoRisk-Client wrapper for 3rd-Party Minecraft launchers written in go";
+    homepage = "https://github.com/technicfan/nrc-wrapper-go/tree/main";
+    license = licenses.gpl3Only;
+    platforms = platforms.all;
+    mainProgram = "nrc-wrapper";
+    maintainers = with maintainers; [mzntori];
+  };
```

```suggestion
  meta = {
    description = "NoRisk-Client wrapper for 3rd-Party Minecraft launchers written in go";
    homepage = "https://github.com/technicfan/nrc-wrapper-go";
    license = lib.licenses.gpl3Only;
    platforms = lib.platforms.all;
    mainProgram = "nrc-wrapper";
    maintainers = with lib.maintainers; [ mzntori ];
  };
```
1. `with` before attrsets (especially `with lib`) was abused to death in the past and is now frowned upon. just prepend `lib.`
2. the maintainers line didn't fit Nixfmt v1's formatting. use it for Nixpkgs contributions